#
# (c) Copyright 2015-2016 River Systems, all rights reserved.
#
# This is proprietary software, unauthorized distribution is not permitted.
#

float32 loadTime
string mapStackFilename
string mapName
string mapVersion
